.circleci/base_config.yml

```
@@ -96,18 +96,21 @@ jobs:
       - run:
           name: Store diff
           command: |
-            git status
-            git -C enterprise status
-            git diff --exit-code > /tmp/clang_format_outcome.txt
-            community=$?
-            git -C enterprise diff --exit-code > /tmp/clang_format_outcome_enterprise.txt
-            enterprise=$?
-            if [ "$community" -ne "0" ] || [ "$enterprise" -ne "0" ]; then
+            OUTCOME_BASE="/tmp/clang_format_outcome"
+            OUTCOME_COMMUNITY=$OUTCOME_BASE/community.diff
+            OUTCOME_ENTERPRISE=$OUTCOME_BASE/community.diff
```

```suggestion
            OUTCOME_ENTERPRISE=$OUTCOME_BASE/enterprise.diff
```

copy and paste type?